health/Cargo.toml

```diff
@@ -15,12 +15,14 @@ readme = "README.md"
 [features]
 default = ["protobuf-codec"]
 protobuf-codec = ["grpcio/protobuf-codec", "protobuf"]
+protobufv3-codec = ["grpcio/protobufv3-codec", "protobufv3"]
 prost-codec = ["grpcio/prost-codec", "prost"]
 
 [dependencies]
 futures-executor = "0.3"
 futures-util = { version = "0.3", default-features = false, features = ["std"] }
-grpcio = { path = "..", version = "0.12.1", default-features = false }
+grpcio = { path = "..", version = "0.12.1", default-features = false, features = ["boringssl"]}
```

Comment:
No, users should decide which ssl to use. `health` is a public crate that is different from benchmark and interop.

Reply:
Reverted, thanks